Только в wget-1.16.1: config.log
Только в wget-1.16.1: config.status
Только в wget-1.16.1/doc: Makefile
Только в wget-1.16.1/doc: wget.1
Только в wget-1.16.1/doc: wget.pod
Только в wget-1.16.1/lib: .deps
Только в wget-1.16.1/lib: accept.o
Только в wget-1.16.1/lib: alloca.h
Только в wget-1.16.1/lib: arg-nonnull.h
Только в wget-1.16.1/lib: arpa
Только в wget-1.16.1/lib: asnprintf.o
Только в wget-1.16.1/lib: base32.o
Только в wget-1.16.1/lib: basename-lgpl.o
Только в wget-1.16.1/lib: binary-io.o
Только в wget-1.16.1/lib: bind.o
Только в wget-1.16.1/lib: c++defs.h
Только в wget-1.16.1/lib: c-ctype.o
Только в wget-1.16.1/lib: charset.alias
Только в wget-1.16.1/lib: cloexec.o
Только в wget-1.16.1/lib: close.o
Только в wget-1.16.1/lib: configmake.h
Только в wget-1.16.1/lib: connect.o
Только в wget-1.16.1/lib: c-strcasecmp.o
Только в wget-1.16.1/lib: c-strncasecmp.o
Только в wget-1.16.1/lib: dirname-lgpl.o
Только в wget-1.16.1/lib: dup2.o
Только в wget-1.16.1/lib: dup-safer.o
Только в wget-1.16.1/lib: dup-safer-flag.o
Только в wget-1.16.1/lib: errno.h
Только в wget-1.16.1/lib: error.o
Только в wget-1.16.1/lib: exitfail.o
Только в wget-1.16.1/lib: fatal-signal.o
Только в wget-1.16.1/lib: fcntl.h
Только в wget-1.16.1/lib: fcntl.o
Только в wget-1.16.1/lib: fd-hook.o
Только в wget-1.16.1/lib: fd-safer.o
Только в wget-1.16.1/lib: fd-safer-flag.o
Только в wget-1.16.1/lib: fnmatch.h
Только в wget-1.16.1/lib: fnmatch.o
Только в wget-1.16.1/lib: fstat.o
Только в wget-1.16.1/lib: futimens.o
Только в wget-1.16.1/lib: gai_strerror.o
Только в wget-1.16.1/lib: getdelim.o
Только в wget-1.16.1/lib: getdtablesize.o
Только в wget-1.16.1/lib: getline.o
Только в wget-1.16.1/lib: getpass.o
Только в wget-1.16.1/lib: getpeername.o
Только в wget-1.16.1/lib: getsockname.o
Только в wget-1.16.1/lib: gettime.o
Только в wget-1.16.1/lib: gettimeofday.o
Только в wget-1.16.1/lib/glthread: .deps
Только в wget-1.16.1/lib/glthread: .dirstamp
Только в wget-1.16.1/lib/glthread: lock.o
Только в wget-1.16.1/lib/glthread: threadlib.o
Только в wget-1.16.1/lib: inet_ntop.o
Только в wget-1.16.1/lib: ioctl.o
Только в wget-1.16.1/lib: langinfo.h
Только в wget-1.16.1/lib: libgnu.a
Только в wget-1.16.1/lib: listen.o
Только в wget-1.16.1/lib: localcharset.o
Только в wget-1.16.1/lib: locale.h
Только в wget-1.16.1/lib: localeconv.o
Только в wget-1.16.1/lib: lseek.o
Только в wget-1.16.1/lib: Makefile
Только в wget-1.16.1/lib: malloc.o
Только в wget-1.16.1/lib: mbchar.o
Только в wget-1.16.1/lib: mbiter.o
Только в wget-1.16.1/lib: mbrtowc.o
Только в wget-1.16.1/lib: mbsinit.o
Только в wget-1.16.1/lib: mbsrtowcs.o
Только в wget-1.16.1/lib: mbsrtowcs-state.o
Только в wget-1.16.1/lib: md5.o
Только в wget-1.16.1/lib: memrchr.o
Только в wget-1.16.1/lib: mkdir.o
Только в wget-1.16.1/lib: mkostemp.o
Только в wget-1.16.1/lib: mkstemp.o
Только в wget-1.16.1/lib: mktime.o
Только в wget-1.16.1/lib: msvc-inval.o
Только в wget-1.16.1/lib: msvc-nothrow.o
Только в wget-1.16.1/lib: netdb.h
Только в wget-1.16.1/lib: netinet
Только в wget-1.16.1/lib: nl_langinfo.o
Только в wget-1.16.1/lib: open.o
Только в wget-1.16.1/lib: pipe2.o
Только в wget-1.16.1/lib: pipe2-safer.o
Только в wget-1.16.1/lib: pipe-safer.o
Только в wget-1.16.1/lib: printf-args.o
Только в wget-1.16.1/lib: printf-parse.o
Только в wget-1.16.1/lib: quotearg.o
Только в wget-1.16.1/lib: raise.o
Только в wget-1.16.1/lib: rawmemchr.o
Только в wget-1.16.1/lib: realloc.o
Только в wget-1.16.1/lib: recv.o
Только в wget-1.16.1/lib: ref-add.sed
Только в wget-1.16.1/lib: ref-del.sed
Только в wget-1.16.1/lib: regex.o
Только в wget-1.16.1/lib: secure_getenv.o
Только в wget-1.16.1/lib: select.o
Только в wget-1.16.1/lib: send.o
Только в wget-1.16.1/lib: setsockopt.o
Только в wget-1.16.1/lib: sha1.o
Только в wget-1.16.1/lib: sigaction.o
Только в wget-1.16.1/lib: sig-handler.o
Только в wget-1.16.1/lib: signal.h
Только в wget-1.16.1/lib: sigprocmask.o
Только в wget-1.16.1/lib: snprintf.o
Только в wget-1.16.1/lib: socket.o
Только в wget-1.16.1/lib: sockets.o
Только в wget-1.16.1/lib: spawn.h
Только в wget-1.16.1/lib: spawn_faction_addclose.o
Только в wget-1.16.1/lib: spawn_faction_adddup2.o
Только в wget-1.16.1/lib: spawn_faction_addopen.o
Только в wget-1.16.1/lib: spawn_faction_destroy.o
Только в wget-1.16.1/lib: spawn_faction_init.o
Только в wget-1.16.1/lib: spawnattr_destroy.o
Только в wget-1.16.1/lib: spawnattr_init.o
Только в wget-1.16.1/lib: spawnattr_setflags.o
Только в wget-1.16.1/lib: spawnattr_setsigmask.o
Только в wget-1.16.1/lib: spawni.o
Только в wget-1.16.1/lib: spawnp.o
Только в wget-1.16.1/lib: spawn-pipe.o
Только в wget-1.16.1/lib: stat.o
Только в wget-1.16.1/lib: stat-time.o
Только в wget-1.16.1/lib: stdio.h
Только в wget-1.16.1/lib: stdio-write.o
Только в wget-1.16.1/lib: stdlib.h
Только в wget-1.16.1/lib: strchrnul.o
Только в wget-1.16.1/lib: strerror.o
Только в wget-1.16.1/lib: strerror_r.o
Только в wget-1.16.1/lib: strerror-override.o
Только в wget-1.16.1/lib: string.h
Только в wget-1.16.1/lib: strings.h
Только в wget-1.16.1/lib: stripslash.o
Только в wget-1.16.1/lib: strndup.o
Только в wget-1.16.1/lib: strnlen1.o
Только в wget-1.16.1/lib: strptime.o
Только в wget-1.16.1/lib: sys
Только в wget-1.16.1/lib: sys_socket.o
Только в wget-1.16.1/lib: tempname.o
Только в wget-1.16.1/lib: time.h
Только в wget-1.16.1/lib: time_r.o
Только в wget-1.16.1/lib: timegm.o
Только в wget-1.16.1/lib: timespec.o
Только в wget-1.16.1/lib: tmpdir.o
Только в wget-1.16.1/lib: unistd.h
Только в wget-1.16.1/lib: unistd.o
Только в wget-1.16.1/lib: unitypes.h
Только в wget-1.16.1/lib/uniwidth: .deps
Только в wget-1.16.1/lib/uniwidth: .dirstamp
Только в wget-1.16.1/lib/uniwidth: width.o
Только в wget-1.16.1/lib: uniwidth.h
Только в wget-1.16.1/lib: utimens.o
Только в wget-1.16.1/lib: vasnprintf.o
Только в wget-1.16.1/lib: vsnprintf.o
Только в wget-1.16.1/lib: waitpid.o
Только в wget-1.16.1/lib: wait-process.o
Только в wget-1.16.1/lib: warn-on-use.h
Только в wget-1.16.1/lib: wchar.h
Только в wget-1.16.1/lib: wctype.h
Только в wget-1.16.1/lib: wctype-h.o
Только в wget-1.16.1/lib: wcwidth.o
Только в wget-1.16.1/lib: write.o
Только в wget-1.16.1/lib: xalloc-die.o
Только в wget-1.16.1/lib: xmalloc.o
Только в wget-1.16.1/lib: xsize.o
Только в wget-1.16.1/lib: xstrndup.o
Только в wget-1.16.1: Makefile
Только в wget-1.16.1/po: Makefile
Только в wget-1.16.1/po: Makefile.in
Только в wget-1.16.1/po: POTFILES
Только в wget-1.16.1/src: .deps
Только в wget-1.16.1/src: build_info.o
Только в wget-1.16.1/src: config.h
Только в wget-1.16.1/src: connect.o
Только в wget-1.16.1/src: convert.o
Только в wget-1.16.1/src: cookies.o
Только в wget-1.16.1/src: css_.o
Только в wget-1.16.1/src: css-url.o
Только в wget-1.16.1/src: exits.o
Только в wget-1.16.1/src: ftp.o
Только в wget-1.16.1/src: ftp-basic.o
Только в wget-1.16.1/src: ftp-ls.o
Только в wget-1.16.1/src: ftp-opie.o
Только в wget-1.16.1/src: hash.o
Только в wget-1.16.1/src: host.o
Только в wget-1.16.1/src: html-parse.o
Только в wget-1.16.1/src: html-url.o
Только в wget-1.16.1/src: http.o
Только в wget-1.16.1/src: http-ntlm.o
Только в wget-1.16.1/src: init.o
Только в wget-1.16.1/src: log.o
Только в wget-1.16.1/src: main.o
Только в wget-1.16.1/src: Makefile
Только в wget-1.16.1/src: mswindows.o
Только в wget-1.16.1/src: netrc.o
Только в wget-1.16.1/src: openssl.o
Только в wget-1.16.1/src: progress.o
Только в wget-1.16.1/src: ptimer.o
Только в wget-1.16.1/src: recur.o
Только в wget-1.16.1/src: res.o
Только в wget-1.16.1/src: retr.o
Только в wget-1.16.1/src: spider.o
Только в wget-1.16.1/src: stamp-h1
Только в wget-1.16.1/src: url.o
diff -urBb wget-1.16.1-orig/src/utils.c wget-1.16.1/src/utils.c
--- wget-1.16.1-orig/src/utils.c	2014-11-23 19:49:06.000000000 +0300
+++ wget-1.16.1/src/utils.c	2015-02-14 21:42:29.656738700 +0300
@@ -64,8 +64,10 @@
 #include <sys/stat.h>
 
 /* For TIOCGWINSZ and friends: */
+#ifndef _WIN32
 #include <sys/ioctl.h>
 #include <termios.h>
+#endif
 
 /* Needed for Unix version of run_with_timeout. */
 #include <signal.h>
Только в wget-1.16.1/src: utils.o
Только в wget-1.16.1/src: version.c
Только в wget-1.16.1/src: version.o
Только в wget-1.16.1/src: warc.o
Только в wget-1.16.1/src: wget.exe
Только в wget-1.16.1/testenv: Makefile
Только в wget-1.16.1/tests: Makefile
Только в wget-1.16.1/util: Makefile
